watertap/unit_models/electrodialysis_1D.py

```diff
@@ -255,15 +316,30 @@ def build(self):
         self.scaling_factor = Suffix(direction=Suffix.EXPORT)
         # Get the base units of measurement from the property definition
         units_meta = self.config.property_package.get_metadata().get_derived_units
+        # Create essential sets
+        add_object_reference(
+            self, "component_set", self.config.property_package.component_list
+        )
+        add_object_reference(
+            self, "solute_set", self.config.property_package.solute_set
+        )
+        add_object_reference(self, "ion_set", self.config.property_package.ion_set)
+
+        add_object_reference(
+            self, "cation_set", self.config.property_package.cation_set
+        )
+        add_object_reference(self, "anion_set", self.config.property_package.anion_set)
+        self.membrane_set = Set(
+            initialize=["cem", "aem"]
+        )  #   cem = Cation-Exchange Membrane aem = Anion-Exchange Membrane
+        self.electrode_side = Set(initialize=["cathode_left", "anode_right"])
```

Comment:
Just being nitpickish here, but how about `electrode_side_set`, i.e., ending the name with `_set` do be consistent with the other set naming convention you have in the model?